packages/cli-platform-ios/src/tools/getArchitecture.ts

```diff
@@ -0,0 +1,14 @@
+import {readFile} from 'fs-extra';
+import path from 'path';
+
+export default async function getArchitecture(iosSourceDir: string) {
```

Comment:
We have the same logic in `doctor` command:
https://github.com/react-native-community/cli/blob/36d00ea428e52d7d45b0cbf36df707825f185317/packages/cli-doctor/src/commands/info.ts#L55-L64

 mind using this function also there - so we won't have in two places same code? 🙏

Reply:
it's already here: https://github.com/react-native-community/cli/blob/ef7a66ab9863ef50ba10b234dc44378d02e0503a/packages/cli-doctor/src/commands/info.ts#L56